### PR TITLE
Update index.md - added RSK Saml

### PR DESCRIPTION
**Pull request by KOdinius**
Added RSK SAML

### CODE REVIEW

File: integrations/index.md [hunk 1]
```diff
@@ -5,4 +5,8 @@ OpenIddict offers native integrations with popular Object–relational Mapping t
...
 - **[P41.OpenIddict.CouchDB](https://github.com/panoukos41/couchdb-openiddict)** by [Panos Athanasiou](https://github.com/panoukos41): CouchDB stores for OpenIddict.
-- **[pixel-identity](https://github.com/Nfactor26/pixel-identity)** by [Nishant Singh](https://github.com/Nfactor26): Ready to host OpenID Connect service using OpenIddict and ASP.NET Identity with a Blazor-based UI for managing users, roles, applications and scopes with support for multiple databases.
\ No newline at end of file
+- **[pixel-identity](https://github.com/Nfactor26/pixel-identity)** by [Nishant Singh](https://github.com/Nfactor26): Ready to host OpenID Connect service using OpenIddict and ASP.NET Identity with a Blazor-based UI for managing users, roles, applications and scopes with support for multiple databases.
+
+Commercially supported components provided by Rock Solid Knowledge:
+
+- **[OpenIddict.SAML2P](https://www.openiddictcomponents.com/?utm_source=openiddictgithubdocumentation&utm_campaign=openiddict)** by [Rock Solid Knowledge](https://www.rocksolidknowledge.com): Ideal for enabling OpenIddict to act as a SAML Identity Provider (IDP) or a SAML Service Provider (SP).
```
Comment (kevinchalet) [inline integrations/index.md]:
```suggestion
Commercially supported components provided by Rock Solid Knowledge:
- **[OpenIddict.SAML2P](https://www.openiddictcomponents.com/?utm_source=openiddictgithubdocumentation&utm_campaign=openiddict)** by [Rock Solid Knowledge](https://www.rocksolidknowledge.com/): Ideal for enabling OpenIddict to act as a SAML Identity Provider (IDP) or a SAML Service Provider (SP).
```

**Review by kevinchalet — COMMENTED** [review]
Thanks for your PR!

**Comment by KOdinius** [comment]
All the changes are done. Thank you for your input! :)

**Comment by kevinchalet** [comment]
Merged, thanks for your PR! ❤️

**Pull request merged**